caliper_sender.py

```diff
@@ -188,18 +219,19 @@ def send_event_batch(batch):
     logger.info("event sent!")
 
 
-def update_runtime_table(): 
+def update_runtime_table(last_event_time): 
     # Insert now into the runtime table after sending event
     now = datetime.utcnow()
     event_time = now.strftime('%Y-%m-%d %H:%M:%S')
     cur.execute("""
-    INSERT INTO cron_run_info (cron_job, last_run_time, last_run_status) 
-    VALUES ('{cron_job}', '{last_run_time}', '{last_run_status}');
+    INSERT INTO cron_run_info (cron_job, last_run_time, last_run_status, last_sent_event_time) 
+    VALUES ('{cron_job}', '{last_run_time}', '{last_run_status}', '{last_sent_event_time}');
     """.format(
         cron_job = 'test_cron', 
         last_run_time = event_time,  
-        last_run_status = 'test_status'))
+        last_run_status = 'test_status',
```

Comment:
This probably was a bug from before but I think this last_run_status should be something like "success". Though we are only inserting the successful runs right now.

Reply:
I can create a new issue for this, should we be logging failures? Hopefully there won't be any! ;) Ok maybe we should log those. . .